File: 백준/Bronze/15552. 빠른 A＋B/빠른 A＋B.py
from sys import stdout, stderr
from __pypy__ import newlist_hint, builders

with open(0, 'r') as f:
    tokens = iter(f.read().split("\n"))
    input = lambda: next(tokens)
    print = lambda *args, sep="\n", end="\n": stdout.write(sep.join(map(str, args)) + end)
    eprint = lambda *args, sep=" ", end="\n": stderr.write(sep.join(map(str, args)) + end)
    is_inbound = lambda pos_x, size_x, pos_y, size_y: 0 <= pos_x < size_x and 0 <= pos_y < size_y
    DELTA = [(-1, 0), (1, 0), (0, -1), (0, 1)]
    INF = 10 ** 18
    MOD = 1_000_000_000
    t = int(input())
    answers = builders.StringBuilder(1 << 18)
    for hh in range(t):
        a, b = map(int, input().split())
        answer = a + b
        for i in f"{answer}":
            answers.append(i)
        answers.append("\n")
    print(answers.build())

File: 백준/Bronze/15552. 빠른 A＋B/빠른 A＋B.kt
import java.io.*
import java.util.*

// Custom input function
val input: () -> String = {
    if (tokens.hasNext()) tokens.next()
    else throw NoSuchElementException("No more tokens available")
}

// Custom print function for standard output
fun print(vararg args: Any?, sep: String = " ", end: String = "\n") {
    val output = args.joinToString(separator = sep) { it.toString() }
    System.out.print(output + end)
}

// Custom eprint function for standard error
fun eprint(vararg args: Any?, sep: String = " ", end: String = "\n") {
    val output = args.joinToString(separator = sep) { it.toString() }
    System.err.print(output + end)
}

// Custom fprint function for file output
fun fprint(vararg args: Any?, sep: String = " ", end: String = "\n", file: PrintWriter) {
    val output = args.joinToString(separator = sep) { it.toString() }
    file.print(output + end)
    file.flush()
}

fun fastParseInt(s: String): Int {
    var num = 0
    var isNegative = false
    for (i in s.indices) {
        val ch = s[i]
        if (i == 0 && ch == '-') {
            isNegative = true
        } else {
            num = (num shl 3) + (num shl 1) + (ch - '0')
        }
    }
    return if (isNegative) -num else num
}

lateinit var tokens: Iterator<String>

fun main() {
    val USE_FILE = false // Set to 'false' to use standard 
    // Set up input stream
    val inputStream: InputStream = if (USE_FILE) {
        File("input2.txt").inputStream()
    } else {
        System.`in`
    }
    val answers = arrayListOf<String>()
    BufferedReader(InputStreamReader(inputStream)).use { reader ->
        var tokenizer = StringTokenizer("")

        val input: () -> String = {
            while (!tokenizer.hasMoreTokens()) {
                val line = reader.readLine() ?: throw NoSuchElementException("No more tokens available")
                tokenizer = StringTokenizer(line)
            }
            tokenizer.nextToken()
        }
        val t = fastParseInt(input())
        val DELTA = arrayListOf(0 to 0, 0 to -1, 0 to 1, -1 to 0, 1 to 0)
        val isInbound =
            { posX: Int, sizeX: Int, posY: Int, sizeY: Int -> (0 <= posX) && (posX < sizeX) && (0 <= posY) && (posY < sizeY) }

        for (h in 1..t) {
            val a = fastParseInt(input())
            val b = fastParseInt(input())
            val answer = a + b
            answers.add("$answer")
        }
    }
    print(*answers.toTypedArray(), sep = "\n")
}